R/vector_control.R

```diff
@@ -155,6 +155,44 @@ throw_away_nets <- function(variables) {
   }
 }
 
+
+
+
+
+#' @title Distribute larval source habitat management
+#' @description simulates larval source management by reducing recruitment to adult female mosquitoes 
```

Comment:
The thinking is that by reducing recruitment to the adult population, we capture any type of the many larval source management strategies that exist. Rather than reflecting a mechanism of action that may be playing out on carrying capacity.